### PR TITLE
Companion for Generate storage info for pallet im_online #9654

### PR DESCRIPTION
**Pull request by georgesdib**
Companion to paritytech/substrate#9654

Adding `MaxKeys`, `MaxPeerInHeartbeats` and `MaxPeerIdEncodingSize` to pallet im_online

@shawntabrizi @KiChjang @thiolliere

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -681,6 +681,9 @@ parameter_types! {
 	pub const BountyValueMinimum: Balance = 200 * CENTS;
 	pub const MaxApprovals: u32 = 100;
 	pub const MaxAuthorities: u32 = 100_000;
+	pub const MaxKeys: u32 = 10_000;
+	pub const MaxPeerInHeartbeats: u32 = 10_000;
+	pub const MaxPeerIdEncodingSize: u32 = 1_000;
```
Comment (thiolliere) [inline runtime/kusama/src/lib.rs:686]: those numbers are probably way bigger than the real limit. But we can make them more precise later.

**Review by thiolliere — APPROVED** [review]
looks good to me

File: runtime/kusama/src/lib.rs (hunk 1 above)
Comment (georgesdib) [reply runtime/kusama/src/lib.rs]: Thank you. Do you mind adding the labels? I don’t have write access to do it myself unfortunately. Thanks!

**Review by KiChjang — APPROVED** [review]

**Comment by KiChjang** [comment]
@georgesdib Do you mind pulling your changes off to another branch instead of working on top of master? I think our CI bot is unable to build this PR due to different master commit histories of your repo vs ours.

**Comment by georgesdib** [comment]
@KiChjang I have rebased and committed from scratch, hope this solves the issue

**Comment by thiolliere** [comment]
bot merge

**Pull request merged**